randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -963,19 +963,31 @@ Extra - minimap_data: [{'x': 70, 'y': 44}, {'x': 70, 'y': 45}, {'x': 70, 'y': 46
   * Extra - instance_id: 134710
   > Door to Bullet Hell Room
       Trivial
+  > EMP Ball
+      Any of the following:
+          Morph Ball and Morph Glide (Advanced)
+          Any of the following:
+              # Shoot the EMP from the door
+              Super Missiles
+              # RNG dependant
+              Missiles and Knowledge (Expert)
 
 > Door to Bullet Hell Room; Heals? False
   * Layers: default
   * Distribution Center Bullet Hell Room Access EMP Door to Bullet Hell Room/Door to Bullet Hell Room Access
   * Extra - instance_id: 134711
-  > Door to Distribution Facility Intersection
+  > Center
       Any of the following:
           Space Jump or Enabled Septogg Helpers or Can Use Spider Ball
-          Speed Booster and Shinesparking Tricks (Beginner)
           # IBJ
           Infinite Bomb Jumping (Expert) and Can Use Bombs
-  > Event - EMP
-      Trivial
+          All of the following:
+              # Shinespark before door to Bullet Hell
+              Speed Booster and Shinesparking Tricks (Beginner)
+              Any of the following:
+                  Short Charge (Intermediate)
+                  # Charge speed from other room
+                  After EMP - Area 5 Bullet Hell Room Access and Disabled Entrance Rando
```

Comment:
this needs disabled door lock rando instead of disabled entrance rando. reaching this node doesnt guarantee that you can go through the door, and every room you can go through should provide you with enough speed, as the difference between short charge and normal is 2 tiles or so.